schemas/common/selfservice.schema.json

```diff
@@ -0,0 +1,170 @@
+          "type": "string",
+          "format": "uri"
+        },
+        "artifact": {
+          "name": "Artifact",
+          "description": "An Artifact use a package forming the implementation of a service. It is is stored in the artifactRepository publised to the artifactRepository by the ciCdPipeline. An artifact has a version number. Once an artifact with a version number is published it becomes immutable, subject to the versioning policy of the artifactRepository.",
+          "type": "string",
+          "format": "uri"
+        },
+
+        "artifactRepository": {
+          "name": "Artifact Repository",
+          "description": "An Artifact repository stores released binaries either from vendors (including Adobe) or created by teh ciCdPipeline. These artifacts are used by the deployment pipleine to deploy an application into an environment. The artifactRepository will allow many versions of an artifact to exist and will be organised in a way that will allow artifacts specific to a tenant, application, topology and environment to be stored. The artifact repsitory must not be used to store any sensitive data.",
+          "type": "string",
+          "format": "uri"
+        },
+        "selfServiceApi": {
+          "name": "Self Service API",
+          "description": "The Self Serivce API is a well defined API that will, using the model defined in XDM expose to a suitably authorised client a service implemenation that allows that client to one or more applications within a tenant. The API must maintain tenant boundaries, as defined and controlled by IMS. The API will expose functionality based on managing a multi tenant application, and some will have access to less functionality (eg a end Customer of Adobe managing their own single tenant deployment.)",
+          "type": "string",
+          "format": "uri"
+        },
+        "selfServicePortal": {
+          "name": "Self Service Portal",
+          "description": "A User interface or potal containg all the functionality of the selfServiceApi presented for use byt a human in a web browsesr interface.",
```

Comment:
When you refer to other properties, it is a good idea to put them in backticks, so that the generated documentation becomes more readable.